src/opentelemetry/lightstep/configuration.py

```diff
@@ -12,78 +12,225 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import logging
-import os
-import sys
-
-import opentelemetry.sdk.trace.propagation.b3_format as b3_format
-from opentelemetry import propagators
-from opentelemetry.lightstep.tracer import configure_tracing
-from opentelemetry.propagators import composite
+from logging import DEBUG, basicConfig, getLogger
+from typing import Optional
+
+from environs import Env
+from grpc import ssl_channel_credentials
+
+from opentelemetry.ext.datadog.propagator import DatadogFormat
+from opentelemetry.lightstep.tracer import LightstepOTLPSpanExporter
+from opentelemetry.lightstep.version import __version__
+from opentelemetry.propagators import set_global_httptextformat
+from opentelemetry.propagators.composite import CompositeHTTPPropagator
+from opentelemetry.sdk.trace import Resource, TracerProvider
+from opentelemetry.sdk.trace.export import (
+    BatchExportSpanProcessor,
+    ConsoleSpanExporter,
+)
+from opentelemetry.sdk.trace.propagation.b3_format import B3Format
+from opentelemetry.trace import get_tracer_provider, set_tracer_provider
 from opentelemetry.trace.propagation.tracecontexthttptextformat import (
     TraceContextHTTPTextFormat,
 )
 
-_ENV_VAR_LS_ACCESS_TOKEN = os.getenv("LS_ACCESS_TOKEN", "")
-_ENV_VAR_LS_SERVICE_NAME = os.getenv("LS_SERVICE_NAME", None)
+_env = Env()
 
-_DEFAULT_SATELLITE_URL = "ingest.lightstep.com:443"
-_ENV_VAR_LS_SATELLITE_URL = os.getenv(
-    "LS_SATELLITE_URL", _DEFAULT_SATELLITE_URL
+_DEFAULT_LS_SATELLITE_URL = "ingest.lightstep.com:443"
+_DEFAULT_LS_METRICS_URL = "ingest.lightstep.com:443/metrics"
+
+_LS_ACCESS_TOKEN = _env.str("LS_ACCESS_TOKEN", None)
+_LS_SATELLITE_URL = _env.str("LS_SATELLITE_URL", _DEFAULT_LS_SATELLITE_URL)
+_LS_METRICS_URL = _env.str("LS_METRICS_URL", _DEFAULT_LS_METRICS_URL)
+_LS_SERVICE_NAME = _env.str("LS_SERVICE_NAME", None)
+_LS_SERVICE_VERSION = _env.str("LS_SERVICE_VERSION", "unknown")
+_LS_PROPAGATOR = _env.list("LS_PROPAGATOR", ["tracecontext", "b3"])
```

Comment:
Only supporting b3 by default for now
```suggestion
_LS_PROPAGATOR = _env.list("LS_PROPAGATOR", ["b3"])
```